server/graphql/v2/interface/Account.ts

```diff
@@ -701,6 +701,9 @@ const accountFieldsDefinition = () => ({
       accountType: {
         type: new GraphQLList(GraphQLAccountType),
       },
+      term: {
```

Comment:
I would enforce `searchTerm` as it's the GraphQL variable name we use for the vendors query, hosted accounts, expenses, and actually most (if not all?) places where we have a similar text search option.

```suggestion
      searchTerm: {
```